cmd/ebrelayer/main.go

```diff
@@ -77,7 +78,7 @@ func initRelayerCmd() *cobra.Command {
 	initRelayerCmd := &cobra.Command{
 		Use:   "init [web3Provider] [contractAddress] [eventSignature] [validatorFromName] --chain-id [chain-id]",
 		Short: "Initializes a web socket which streams live events from a smart contract",
-		Args:  cobra.ExactArgs(4),
+		Args:  cobra.MinimumNArgs(4),
```

Comment:
undo, we should always provide exact args. If there's an optional arg we can use a flag instead for it. We'd also need to add a `Long:` description to provide a concrete example for using the flag.